src/components/SpendInput.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const SpendInput = () => {
```

Comment:
As discussed @tams2429 lets rename this to `Spending` to future proof the apps naming convention

Reply:
Yep will change this in the upcoming commit, and will push for review